src/data/pengestopp/flaggperson_hooks.ts

```diff
@@ -0,0 +1,22 @@
+import { useSelector } from "react-redux";
+import { RootState } from "../rootState";
+
+export const useIsStopped = (): boolean => {
+  const flaggperson = useSelector((state: RootState) => state.flaggperson);
+
+  return (
+    flaggperson.endret && !flaggperson.endrer && !flaggperson.endringFeilet
+  );
+};
+
+export const useHasServerErrors = (): boolean => {
```

Comment:
Kan denne hete noe med `flaggperson`, så vi vet hva som har feilet? Jeg føler det blir litt lite intuitivt når man kun skriver `useHasServerErrors` et annet sted i koden. Eventuelt det samme for `useIsStopped`, selv om det er litt enklere å skjønne.

Reply:
Hadde det vært bedre med én hook (useFlaggperson) som returnerte de tre tilstands-verdiene?

Reply:
Gode innspill. Fikser!